frontend/public/components/utils/dropdown.jsx

```diff
@@ -92,8 +92,15 @@ const Caret = () => <span className="caret" />;
 
 class DropDownRow extends React.PureComponent {
   render() {
-    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey} = this.props;
+    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey, autocompleteFilter} = this.props;
     let prefix;
+
+    if(!autocompleteFilter){
+      //use pf4 markup if not using the autocomplete dropdown
+      return <li key={itemKey}>
+        <button className="pf-c-dropdown__menu-item" id={`${itemKey}-link`} onClick={e => onclick(itemKey, e)}>{content}</button>
```

Comment:
This probably will break enter key usage in most forms. Add `type="button"` so it's never treated as a submit button. We might need prevent default on the click event.